platform/broadcom/sonic-platform-modules-dell/s5248f/setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup
+
+setup(
+    name='sonic-platform',
+    version='1.0',
+    description='SONiC platform API implementation on DellEmc Platforms',
+    license='Apache 2.0',
+    author='SONiC Team',
+    author_email='linuxnetdev@microsoft.com',
+    url='https://github.com/Azure/sonic-buildimage',
+    maintainer='DellEMC',
+    maintainer_email='dell-sonic@dell.com',
+    packages=[
+        'sonic_platform',
+    ],
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Plugins',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: Apache Software License',
+        'Natural Language :: English',
+        'Operating System :: POSIX :: Linux',
+        'Programming Language :: Python :: 2.7',
```

Comment:
Again, suggest removing Python 2 support for the sonic-platform package. Please verify everything works as expected with only the Python 3 package on the 202012 branch.

Reply:
Removed the Python2 support and tested with Python3 package.